iac-extensions/spring-config/src/main/java/org/sonar/iac/springconfig/plugin/SpringConfigExtension.java

```diff
@@ -23,13 +23,17 @@
 import org.sonar.api.Plugin;
 
 public final class SpringConfigExtension {
+  public static final String SENSOR_NAME = "Java Config Sensor";
+  public static final String LANGUAGE_KEY = "java";
+  public static final String LANGUAGE_NAME = "Java";
   public static final String REPOSITORY_KEY = "javaconfig";
 
   private SpringConfigExtension() {
   }
 
   public static void define(Plugin.Context context) {
     context.addExtensions(List.of(
+      SpringConfigSensor.class,
```

Comment:
I know we don't have any checks right now, but shouldn't the `SpringCOnfigRulesDefinition` also be registered here?